### PR TITLE
[Tutorial] Deploy an Edge Cluster on VirtualBox

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR adds the Deploy an Edge Cluster on VirtualBox tutorial.

Note: this tutorial will be updated with the latest version of CanvOS after the release 4.4.b.

## Changed Pages

💻 [Deploy an Edge Cluster on VirtualBox](https://deploy-preview-3561--docs-spectrocloud.netlify.app/tutorials/edge/deploy-cluster-virtualbox/)

## Jira Tickets

🎫 [DOC-1185](https://spectrocloud.atlassian.net/browse/DOC-1185)

## Backports

Can this PR be backported?

- [X] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1185]: https://spectrocloud.atlassian.net/browse/DOC-1185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 1]
```diff
@@ -0,0 +1,725 @@
...
+![A screenshot of the VirtualBox VM after installation.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_vm-reboot.webp)
+
+Next, click **Settings** and select **Storage**.
+
+Select the Edge Installer ISO and click **Remove Attachment** to remove it from your VM. Confirm the deletion with
+**Remove** and click **OK** to close the settings window.
+
+![A screenshot of the VirtualBox VM storage configuration.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_vm-remove-iso.webp)
+
+### Deploy Edge Host
```
Comment (lennessyy) [inline docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:552]:
```suggestion
### Register Edge Host
```
I think using Register is more specific here

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 2]
```diff
@@ -0,0 +1,725 @@
...
+tag **env:edge**. Click **Next**.
+
+Select the cluster profile you created in the [Create Cluster Profile](#create-cluster-profile) section of this tutorial
+and click **Confirm**.
+
+The **Cluster Profile** section displays all the layers in the cluster profile. Click on **Next** to proceed.
+
+![A screenshot of the cluster profile during cluster deployment.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_cluster-deployment-profile.webp)
+
+In the **Cluster Config** section, provide a Virtual IP (VIP) address for the Edge cluster. This address must be an
```
Comment (lennessyy) [inline docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:592]: In the context of virtual box, maybe a little help in identifying the network that the VM is on?

**Review by lennessyy — APPROVED** [review]
Looks great! It's very nice that users can now play with Edge without having a whole vCenter

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 3]
```diff
@@ -0,0 +1,725 @@
...
+
+```shell
+git tag
+```
+
+The output should be similar to the following.
+
+<!-- REPLACE 4.4.6 -->
+
+```text hideClipboard
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:113]: oh, yes. I agree :)

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md (hunk 2 above)
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:592]: huum. The VM will be in the same network as the host machine, as mentioned in the `Deploy VirtualBox VM` section. I added a tip to help the user scan the network and check the used IP addresses.

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 4]
```diff
@@ -0,0 +1,683 @@
...
+Confirm your Edge host is listed as **Healthy** and with a **Ready** status. The **Machine ID** should match the ID
+displayed on your VM's screen.
+
+![A screenshot of the Edge host in Palette.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_edge-host-palette.webp)
+
+## Deploy Edge Cluster
+
+The following steps will guide you through deploying the Edge cluster.
+
+Select **Clusters** from Palette's left **Main Menu** and click on **Create Cluster**. Ensure you are in the **Default**
```
Comment (addetz) [inline docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:526]:
```suggestion
Select **Clusters** from the left **Main Menu** and click on **Create Cluster**. Ensure you are in the **Default**
```

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 5]
```diff
@@ -0,0 +1,683 @@
...
+
+In this tutorial, you successfully deployed a single-node Edge cluster along with a demo application using a VirtualBox
+VM as the Edge host. You learned how to build and test Edge artifacts, prepare an Edge host, and use it to deploy an
+Edge cluster.
+
+Palette Edge enables you to customize your Edge hosts with the desired OS, Kubernetes distribution, dependencies, and
+user data configurations.
+
+This tutorial has provided you with hands-on experience with Palette Edge using a single VM, eliminating the need for a
+complex lab environment or separate physical devices. You can also use this setup to quickly test and validate Edge
```
Comment (addetz) [inline docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:676]:
```suggestion
complex lab environment or separate physical devices. You can also use this setup to test and validate Edge
```

**Review by addetz — APPROVED** [review]
Wonderful work! 🚀  Just left you some small suggestions, but the tutorial is really well structured and easy to follow.

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 6]
```diff
@@ -0,0 +1,683 @@
...
+
+:::tip
+
+If you want your Edge host to have preloaded content and be able to create clusters using this content, you can create a
+[content bundle](../../clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md) and build it into your
+Installer ISO.
+
+:::
+
+### Check Out the Starter Code
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:87]: Loved it!

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 7]
```diff
@@ -0,0 +1,683 @@
...
+
+Palette Edge allows the deployment of Kubernetes workloads in remote locations with limited connectivity and compute
+infrastructure. This means you can use Palette to manage the lifecycle of your Kubernetes clusters at the edge in places
+such as hospitals, rural areas, restaurants, and more.
+
+Edge clusters are Kubernetes clusters set up on Edge hosts, which can be bare metal or virtual machines. These hosts can
+be managed locally on-site through the [Local UI](../../clusters/edge/local-ui/local-ui.md) or centrally through the
+Palette management plane.
+
+Before forming a cluster, the Edge Hosts must be prepared and registered with Palette. This involves the
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md]: Soo, Lenny and I actually talked about this word before, and we both agreed it makes sense :sweat_smile:

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 8]
```diff
@@ -0,0 +1,683 @@
...
+[Provider Images](../../clusters/edge/edgeforge-workflow/palette-canvos/build-provider-images.md). Once these artifacts
+are built, you can use the Installer ISO to bootstrap the Edge installation on your Edge host and the Provider Images to
+create a cluster profile.
+
+This tutorial will help you understand, through hands-on activities, how the different Edge components work together.
+You will build and test the Edge artifacts and deploy an Edge cluster without needing a complex lab environment or
+separate physical devices. Specifically, you will learn to deploy an Edge cluster using a VirtualBox VM as the Edge
+host. The tutorial will guide you through building the required
+[Edge artifacts](../../clusters/edge/edgeforge-workflow/palette-canvos/palette-canvos.md), creating an Edge cluster
+profile, preparing your Edge host, and deploying an Edge cluster along with a demo application.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:35]: I agree! Removed!

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 9]
```diff
@@ -0,0 +1,683 @@
...
+**Edge Hosts** tab to view the registered hosts.
+
+Confirm your Edge host is listed as **Healthy** and with a **Ready** status. The **Machine ID** should match the ID
+displayed on your VM's screen.
+
+![A screenshot of the Edge host in Palette.](/tutorials/edge-vbox/tutorials_edge-vbox_deploy-cluster-virtualbox_edge-host-palette.webp)
+
+## Deploy Edge Cluster
+
+The following steps will guide you through deploying the Edge cluster.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:522]: True! Removed :)

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 10]
```diff
@@ -0,0 +1,683 @@
...
+### Install Palette Edge
+
+In VirtualBox, select the created VM and click **Start** to turn it on. The Edge Installer will bootstrap the Palette
+Edge installation onto the VM.
+
+Wait for the Edge Installer to complete copying content to the VM, which may take a few minutes. The VM will reboot by
+default upon completion.
+
+When the image below appears, right-click the VM, select **Stop**, and then click **Power Off** to turn it off before
+the reboot starts.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:489]: Yes! Done :)

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 11]
```diff
@@ -0,0 +1,683 @@
...
+Once the provider images are available in the registry, create the cluster profile.
+
+Log in to [Palette](https://console.spectrocloud.com/) and select **Profiles** from the left **Main Menu**. Click **Add
+Cluster Profile** to create a cluster profile.
+
+Follow the wizard to create a new profile.
+
+In the **Basic Information** section, assign the name **edge-vbox-profile** and a brief profile description, select the
+type as **Full**, and assign the tag **env:edge**. You can leave the version empty if you want to. Just be aware that
+the version defaults to **1.0.0**. Click on **Next**.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:353]: I tried adding the table, but i think I prefer the text in this case :(

File: docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md [hunk 12]
```diff
@@ -0,0 +1,683 @@
...
+The build may take 15 to 20 minutes to conclude, depending on the hardware resources available on the host machine. Once
+finished, you get a success message similar to the one displayed below.
+
+```text hideClipboard
+# Output condensed for readability
+========================== 🌍 Earthly Build  ✅ SUCCESS ==========================
+🛰️ Reuse cache between CI runs with Earthly Satellites! 2-20X faster than without cache. Generous free tier https://cloud.earthly.dev
+```
+
+The output also includes a manifest with predefined parameters you will use later to create the cluster profile. Copy
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/tutorials/edge/deploy-cluster-virtualbox.md:263]: I rephrased it so that it is clear the user will need this later
